app/main.py

```diff
@@ -1,13 +1,27 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error.get("line_number"),
+            "column": error.get("column_number"),
+            "message": error.get("text"),
+            "name": error.get("code"), "source": "flake8"
+            }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "path": file_path,
+        "errors": [format_linter_error(i) for i in errors],
+        "status": "failed" if any(errors) else "passed"
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        {"errors": [],
+         "path": "./test_source_code_2.py",
```

Comment:
why do you use hard code? you can get path from `linter_report`

Reply:
"path": linter_report.get("./test_source_code_2.py") - changed 
But I do not know if it right